ppb_vector/vector2.py

```diff
@@ -77,6 +78,8 @@ def __init__(self, x: typing.SupportsFloat, y: typing.SupportsFloat):
         except ValueError:
             raise TypeError(f"{type(y).__name__} object not convertable to float")
 
+    update = dataclasses.replace
```

Comment:
How does this look in `help()`/pydoc?

Reply:
```python3
>>> from ppb_vector import Vector2
>>> help(Vector2.update)
Help on function replace in module dataclasses:

replace(obj, **changes)
    Return a new object replacing specified fields with new values.
    
    This is especially useful for frozen classes.  Example usage:
    
      @dataclass(frozen=True)
      class C:
          x: int
          y: int
    
      c = C(1, 2)
      c1 = replace(c, x=3)
      assert c1.x == 3 and c1.y == 2
```

That's indeed not very nice.  OTOH, `help()` is currently useless on `Vector2` because:
- Type constructs like `VectorLike` are expanded into an illegible mess.
- Description & documentation are in the `README` rather than in docstrings.

I think it would be worth fixing, but it's way out of scope for this PR.

Reply:
Fair enough. We should figure out a way to handle that.